src/components/ec-table-filter/ec-table-filter.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <section
+    class="ec-table-filter"
+    data-test="ec-table-filter"
+  >
+    <component
+      :is="filter.component"
+      v-for="(filter, index) in filters"
+      :key="index"
+      :value="value[filter.name]"
+      v-bind="filter"
```

Comment:
we need to remove the component from the v-bind, otherwise it renders:
```
<div data-test="ec-table-filter__filter-item-1" class="ec-sync-multiple-values-filter ec-table-filter__filter-item" name="feeType" component="[object Object]">
```

do instead
```
v-bind="{ ...filter, component: null }"
```

BTW, I noticed that the data-test from this component overrides data-test from the filter component, we should then add 
```
:data-test="$attrs['data-test'] ? `${$attrs['data-test']} ec-sync-multiple-values-filter` : 'ec-sync-multiple-values-filter'"
```
to `ec-sync-multiple-values-filter` and fix it in ec-date-range-filter too.

Reply:
👍